### PR TITLE
add binding for ErgoBox in wasm

### PR DESCRIPTION
**Pull request by minotaur-ergo**

### CODE REVIEW

File: bindings/ergo-lib-wasm/package.json [hunk 1]
```diff
@@ -27,5 +27,7 @@
         "webpack-cli": "^3.1.1",
         "webpack-dev-server": "^3.1.0"
     },
-    "dependencies": {}
+    "dependencies": {
+        "wasm-pack": "^0.10.1"
```
Comment (greenhat) [inline bindings/ergo-lib-wasm/package.json:31]: Why it's needed here? I mean it seems like a wrapper for a wasm-pack binary from what I see in https://github.com/rustwasm/wasm-pack/tree/master/npm

**Review by greenhat — CHANGES_REQUESTED** [review]
Looking good! See my question on adding a dependency.

File: bindings/ergo-lib-wasm/package.json (hunk 1 above)
Comment (minotaur-ergo) [reply bindings/ergo-lib-wasm/package.json]: It is required for all build processes locally. as you can see in your package.json for example build-browser command used wasm-pack to build project.
my mistake is to install it as dependency. but it is a devDependency.

Comment (minotaur-ergo) [reply bindings/ergo-lib-wasm/package.json]: I think Its required to to installed as devDependency to build project without GA. but removed it from project for now.

**Review by greenhat — APPROVED** [review]
Great! Thank you!

**Pull request merged**